include/mapnik/raster.hpp

```diff
@@ -40,15 +40,27 @@ class raster : private util::noncopyable
 {
 public:
     box2d<double> ext_;
+    box2d<double> query_ext_;
```

Comment:
No, because we need to know when it stops on the other side as well, so we need to know two full extents.